src/main/java/com/possible/dhis2int/db/DatabaseDriver.java

```diff
@@ -173,4 +173,38 @@ public void dropImamTable() throws DHISIntegratorException {
 		}
 	}
 
+	public void createTempFamilyTable(Integer numberOfVasectomyUser, Integer numberOfPillsUser, Integer numberOfOtherUser, Integer numberOfMinilipUser,
+									  Integer numberOfIUCDUser,Integer numberOfImplantUser,Integer numberOfDepoUser,Integer numberOfCondomsUser
+	) throws DHISIntegratorException {
+		logger.info("create family planning temp table method.");
+
+		Connection connection = null;
+		try {
+			connection = DriverManager.getConnection(properties.openmrsDBUrl);
+			Statement statement = connection.createStatement();
+			statement.executeUpdate("DROP TABLE IF EXISTS familyPlanning");
+			statement.executeUpdate(
+					"CREATE TABLE familyPlanning(vasectomy_user int, pills_user int, other_user int, minilap_user int,IUCD_user int, implant_user int, depo_user int, condoms_user int)");
+
+			String insertFamilyPlanningData = new StringBuffer(
+					"INSERT INTO familyPlanning(vasectomy_user, pills_user, other_user,  minilap_user, IUCD_user, implant_user, depo_user, condoms_user) SELECT ")
+					.append(numberOfVasectomyUser).append(", ").append(numberOfPillsUser).append(", ")
+					.append(numberOfOtherUser).append(", ").append(numberOfMinilipUser).append(", ")
+					.append(numberOfIUCDUser).append(", ").append(numberOfImplantUser).append(", ")
+					.append(numberOfDepoUser).append(", ").append(numberOfCondomsUser).toString();
+			statement.executeUpdate(insertFamilyPlanningData);
+
+		} catch (SQLException e) {
+			throw new DHISIntegratorException(String.format("Failed to create table familyPlanning"), e);
```

Comment:
Trace this exception and make sure this exception is not swallowed somewhere as it seems its not handled from its callers, Try replicating the error for this.